CHANGELOG.md

```diff
@@ -1,7 +1,52 @@
 # Changelog
 All notable changes to Ergo will be documented in this file.
 
-## [2.9.1] - 2021-01-10
+## [2.10.0-rc1] - 2022-05-15
+
+We're pleased to be publishing the release candidate for 2.10.0 (the official release should follow in a week or so).
+
+This release contains no changes to the config file format or database file format.
+
+Many thanks to [@csmith](https://github.com/csmith), [@FiskFan1999](https://github.com/FiskFan1999), [@Mikaela](https://github.com/Mikaela), [@progval](https://github.com/progval), and [@thesamesam](https://github.com/thesamesam) for contributing patches, and to [@emersion](https://github.com/emersion), [@eskimo](https://github.com/eskimo), [@FiskFan1999](https://github.com/FiskFan1999), [@jigsy1](https://github.com/jigsy1), [@Mikaela](https://github.com/Mikaela), [@mogad0n](https://github.com/mogad0n), [@progval](https://github.com/progval), and [@xnaas](https://github.com/xnaas) for reporting issues and helping test.
+
+### Config changes
+
+* For better interoperability with [Goguma](https://sr.ht/~emersion/goguma/), the recommended value of `history.chathistory-maxmessages` has been increased to `1000` (previously `100`) (#1919)
+
+### Changed
+* Persistent voice (`AMODE +v`) in a channel is now treated as a permanent invite (i.e. overriding `+i` on the channel) (#1901, thanks [@eskimo](https://github.com/eskimo)!)
```

Comment:
This doesn't seem to be in the documentation yet?

Reply:
You're saying, it should be documented in the section of the manual that describes `+i`?

Reply:
Oh, never mind, I see what you did over on #1959